app/presenters/supergroups/supergroup.rb

```diff
@@ -53,5 +53,9 @@ def document_list(taxon_id)
         data
       end
     end
+
+    def special_format_count(*)
+      0
```

Comment:
Can we set '3' as the default, so we don't have to override this method in every supergroup class?